lib/ci/directbindingarray.rb

```diff
@@ -21,6 +21,7 @@ def self.to_volumes(array)
 
     # @return [Array] Binds API array of the form ["Path:Path"]
     def self.to_bindings(array)
+      array ||= []
```

Comment:
one would argue that this method should not be called with anything but an array and if one calls it with anything but an array that is a bug in the calling code

also if anything the code should read `return [] unless array` as this very line is in a very explicit fashion rendering the collect {} noop, as such initializing it to empty and then running collect on an empty array collecting another empty array is bullshit of the highest order